tests/plugins/balloontoolbar/manual/divareascroll.html

```diff
@@ -0,0 +1,142 @@
+<style>
+	body {
+		height: 5000px;
+	}
+	.top-margin {
+		margin-top: 500px;
+	}
+</style>
+<h2 class="top-margin">Divarea:</h2>
+<div contenteditable="true" id="editor1" class="editor">
```

Comment:
I really like the simplicity of this manual test. I'm only wondering if we could avoid paragraph duplication by using proper CSS for that, something which has been done for autocomplete tests: https://github.com/ckeditor/ckeditor4/blob/master/tests/plugins/autocomplete/manual/viewposition.html